src/control/group/mod.rs

```diff
@@ -0,0 +1,35 @@
+cfg_if! {
```

Comment:
This is just copied directly from the `raw` module, without any additional changes.